scripts/install

```diff
@@ -55,7 +55,7 @@ ynh_app_setting_set --app=$app --key=is_public --value=$is_public
 #=================================================
 ynh_script_progression --message="Installing dependencies..." --weight=25
 
-ynh_install_app_dependencies libav-tools php-cli
+ynh_install_app_dependencies ffmpeg php-cli
```

Comment:
Again if we want to keep backward compatibility, I believe we should use the 'or' syntax here, I believe it's like this but not sure .. :

 ```suggestion
ynh_install_app_dependencies "libav-tools|ffmpeg" php-cli
```